### ISSUE TITLE
addInstrumentName -> track name?

### ISSUE THREAD

**Issue by revyTH** [grimmdude/MidiWriterJS#28]
Hi, thanks for this useful package!

I've some problems to set tracks name. I've tried the following:

```
var tracks = [];
tracks[0] = new MidiWriter.Track();
tracks[0].addEvent(new MidiWriter.ProgramChangeEvent({instrument : 1}));
tracks[0].addInstrumentName("piano");
```

but when importing in Ableton or converting as json at [https://tonejs.github.io/MidiConvert/](https://tonejs.github.io/MidiConvert/) the tracks name doesn't appear. Any hint?

Thanks, cheers!

**Comment by grimmdude** [comment]
Hi There,

I believe it will depend on the MIDI software whether or not the `Instrument Name` event overrides the General Midi instrument name associated with the program number.  In most cases I've found it doesn't.  Here is a list of GM sounds and associated numbers: 

https://www.midi.org/specifications/item/gm-level-1-sound-set

There is a `Sequence/Track Name` event in the MIDI spec which sounds like it may be more suited to what you're looking for.  I've just added a method for this (`Track.addTrackName`), however it doesn't seem to be recognized by that online MIDI parser either.  You might have better luck with that in Ableton though.

-Garrett
